frames/src/test/java/com/checkout/frames/screen/billingaddressdetails/usecase/BillingAddressInputComponentStyleUseCaseTest.kt

```diff
@@ -84,16 +88,16 @@ internal class BillingAddressInputComponentStyleUseCaseTest {
                     infoStyle = TextLabelViewStyle(),
                     inputFieldStyle = InputFieldViewStyle(),
                     subtitleStyle = TextLabelViewStyle(),
-                    titleStyle = TextLabelViewStyle()
-                )
+                    titleStyle = TextLabelViewStyle(),
+                ),
 
-            )
+                )
```

Comment:
Empty space, worth adding a check for it

Reply:
Hmmm looks like we don't have this option :(